api/services/elastigroup/aws/paths/elastigroup-deployment.yaml

```diff
@@ -35,36 +35,3 @@ put:
       $ref: "../responses/group-deployment-stop.yaml"
     400:
       description: "Bad Request"
-
-get:
```

Comment:
If this is removed, and the file only contains stop deployment, we better rename the file to elastigroup-stop-deployment